### ISSUE TITLE
how to get and test the ZH-EN results

### ISSUE THREAD

**Issue by bozhenhhu** [eric-xw/Video-guided-Machine-Translation#11]
Dear @eric-xw @zzxslp 
    Thanks for your great work. Based on your model, I want to get the ZH-EN results, how can I do it? just change this line 
    src, tgt = 'en', 'zh' to  src, tgt = 'zh', 'en'？However, If I train  twice to get the en-zh and zh-en results, this model woud not like other MT models  that can get the two kinds of  results with training only once. 
The second question is that **assuming I have got the ZH-EN submisson file on the public test dataset, how can I test my results?** I find that this challenge page(https://competitions.codalab.org/competitions/24384) **may** only test the EN-ZH results. So I am very confused to get the ZH-EN BLEU-4：24.6 ？
![image](https://user-images.githubusercontent.com/37445920/89505777-b9f82800-d7fc-11ea-92c2-2277e30eae47.png)

**Comment by eric-xw** [comment]
Hi, we are holding out the Chinese captions of the test set for challenge use, so the ZH->EN results cannot be evaluated on the test set. But you may evaluate the EN->ZH results on the test set or the full results on the validation set.
